sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/BitwiseExpressionsSuite.scala

```diff
@@ -131,4 +132,25 @@ class BitwiseExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       checkConsistencyBetweenInterpretedAndCodegen(BitwiseXor, dt, dt)
     }
   }
+
+  test("BitGet") {
```

Comment:
The `checkConsistencyBetweenInterpretedAndCodegen` should also be used:
```
    DataTypeTestUtils.integralType.foreach { dt =>
      checkConsistencyBetweenInterpretedAndCodegen(BitwiseXor, dt, dt)
    }
```

Reply:
The implement of BitwiseGet cannot test with the utils

Reply:
Why is that? It should work for almost all expressions

Reply:
`Analyzer` call the `TypeCoercion` and the latter  cast byte|short to integer.
checkConsistencyBetweenInterpretedAndCodegenAllowingException cannot uses `Analyzer`, so the BitwiseGet get the byte as input and throw `java.lang.ClassCastException: java.lang.Byte cannot be cast to java.lang.Integer`.

Reply:
Yes, this snipet does not work if you support only integer inputs.
`checkConsistencyBetweenInterpretedAndCodegenAllowingException(BitwiseGet(_, _), IntegerType, IntegerType)` should work in the current code.

Reply:
OK